src/main/java/com/exasol/adapter/dynamodb/queryrunner/PlanDoesNotFitException.java

```diff
@@ -0,0 +1,12 @@
+package com.exasol.adapter.dynamodb.queryrunner;
+
+/**
+ * This Exception is thrown by a QueryPlanFactory if the plan does not fit for the given query.
+ */
+public class PlanDoesNotFitException extends Exception {
+    private static final long serialVersionUID = -3257463856415639740L;
+
+    PlanDoesNotFitException(final String message) {
```

Comment:
```suggestion
    public PlanDoesNotFitException(final String message) {
```

Reply:
I made the constructor packat private as the Exception shall not be thrown from outside this package